setup.py

```diff
@@ -29,6 +29,11 @@
     pip install -e .[test]
     pytest
 
+or::
+
+    python setup.py build_ext --inplace
```

Comment:
This is not completely equivalent to `pip install -e .[test]`, is it? Does it install the dependencies listed under `test` section in `setup.cfg`?

Reply:
This does not install the package, it just builds extensions. `setup.py develop` is closer (and run `build_ext`) but it will not install test dependencies.

Reply:
Might be confusing to just "or" without any explanation here. When do I know which command I should use?

Reply:
Shouldn't we have fewer invocations of `setup.py`, and not more? I'm wondering why a user would avoid installing the development version (the advice could be: "if you don't want to overwrite your perfectly stable Astropy version, create a new environment for development").

Reply:
I am hoping that by the time they resort to installing from source, they already have a dev workflow and know what they are doing. If we start to be specific, then we have to cover all the different environment managers and dev workflows that one might be using when they encounter this text. It is a slippery slope.